### ISSUE TITLE
[Bug]: `link-color` not being used for the About Page links

### ISSUE THREAD

**Issue by kommunarr** [FreeTubeApp/FreeTube#7174]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

From https://github.com/FreeTubeApp/FreeTube/issues/6472#issuecomment-2565695788: We should switch to the link-color for the About page links in the same effort due to it being the only apparent instance of mixing the theme colors with a bg-color background, thus failing color contrast issues. Such a fix should check that this succeeds in achieving a sufficient color contrast in this case, otherwise another solution will have to be pursued.

### Expected Behavior

`link-color` is being used on the About page links and there are no insufficient color contrast issues present

### Issue Labels

visual bug, accessibility issue

### FreeTube Version

v0.23.3

### Operating System Version

OpenSUSE TW

### Installation Method

.rpm

### Primary API used

Local API

### Nightly Build

- [x] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by ilysuha** [comment]
This fix in another issue and it might help you [click](https://newrides.es/captha/)
